packages/grafana-ui/src/components/Typeahead/PartialHighlighter.test.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
```

Comment:
This is more of a question for you than suggestion (learning moment for me 🙂). We are generally trying to move from snapshot testing. But for some cases snapshot tests are better than writing unit test. So I am wondering, if you would say this is a case. Cause I would personally do unit test using react testing library and check it correctly highlights text. But wondering if snapshot makes more sense as this component would rarely change.

Reply:
I find it simpler to use when there's a complex rendering logic based on properties as it becomes simple input/output testing. At the same time I prefer consistent approach to testing so if snapshots are *passé* I'm happy to covert it to a assertions :)

Reply:
I would personally think that assertions are more consistent.  But this is definitely non-blocker.

Reply:
I've converted the test to assertions.